### PR TITLE
Publisher Welcome Page UI Improvements

### PR DESCRIPTION
**Pull request by Sarangan0219**
This PR will fixes https://github.com/wso2/product-apim/issues/6672 and fixes https://github.com/wso2/product-apim/issues/6676

- Before Creating any APIs

Screenshot from 2020-03-02 14-08-14

- After creating an API

Screenshot from 2020-03-02 14-08-31

This consistency is followed in creating API Products too.

And Enabling the Sample API Product deployment,

- Before Creating any API Products.

Screenshot from 2020-03-02 14-11-04

- After Deploying the Sample API Product

Screenshot from 2020-03-02 14-11-23

### CODE REVIEW

File: features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Listing/SampleAPI/SampleAPIProduct.jsx [hunk 1]
```diff
@@ -169,21 +175,32 @@ function SampleAPI(prop) {
...
                 <div className={classes.actions}>
+                    <APIProductCreateMenu buttonProps={{
+                        size: 'small',
+                        color: 'primary',
+                        variant: 'contained',
+                        className: classes.buttonLeft,
+                    }}
+                    >
+                        <Create />
+                        <FormattedMessage id='Create.New.Api.Product' defaultMessage='Create New API Product' />
```
Comment (menakaj) [inline features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Listing/SampleAPI/SampleAPIProduct.jsx]: Use the proper format of id. 

The Id should be `Apis.Listing.SampleAPI.SampleAPIProduct.create.new.api.product`

Comment (menakaj) [reply features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Listing/SampleAPI/SampleAPIProduct.jsx]: <Relative_path_of_the_file>.message.content

File: features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Listing/components/APIProductCreateMenu.jsx [hunk 2]
```diff
@@ -0,0 +1,50 @@
+import React from 'react';
```
Comment (menakaj) [inline features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Listing/components/APIProductCreateMenu.jsx]: Add license header.

Comment (Sarangan0219) [reply features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Listing/components/APIProductCreateMenu.jsx]: https://github.com/wso2/carbon-apimgt/pull/8279/commits/898e7283d2ef894f03b48909bef6d76165a42dc8

File: features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Listing/SampleAPI/SampleAPIProduct.jsx (hunk 1 above)
Comment (Sarangan0219) [reply features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Listing/SampleAPI/SampleAPIProduct.jsx]: https://github.com/wso2/carbon-apimgt/pull/8279/commits/898e7283d2ef894f03b48909bef6d76165a42dc8

File: features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Listing/components/APICreateMenu.jsx [hunk 3]
```diff
@@ -1,3 +1,21 @@
+/*
+ * Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (menakaj) [inline features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Listing/components/APICreateMenu.jsx]: Year should be 2020

File: features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Listing/components/APIProductCreateMenu.jsx [hunk 4]
```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (menakaj) [inline features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Listing/components/APIProductCreateMenu.jsx]: Change year

File: features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Listing/components/APICreateMenu.jsx (hunk 3 above)
Comment (Sarangan0219) [reply features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Listing/components/APICreateMenu.jsx]: https://github.com/wso2/carbon-apimgt/pull/8279/commits/4c0c10bd001c827254f1e9958dc6ca007f4ef4c3

File: features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Listing/components/APIProductCreateMenu.jsx (hunk 4 above)
Comment (Sarangan0219) [reply features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Listing/components/APIProductCreateMenu.jsx]: https://github.com/wso2/carbon-apimgt/pull/8279/commits/4c0c10bd001c827254f1e9958dc6ca007f4ef4c3

**Pull request merged**